en/editor/assets/index.md

```diff
@@ -111,6 +111,28 @@ The search function is a combined function that can limit the search type and sp
 - Shortcut key `Esc` to cancel rename
 - In addition, the initial name of the Typescript asset will be treated as its `className`, and the `className` cannot be repeated.
 
+### Exporting Assets Package
+
+Select the resource to be exported, right-click on it, then select **Export Resource Package**. The **Export Resource** panel will automatically list the currently selected resource and its dependencies. To not export the dependent resources, uncheck **Include Dependent Resources** n the bottom left corner of the **Export Resources** panel.
+
+Once identified, the resources can be exported by clicking the **Export** button to bring up the file storage dialog box where it is necessary to specify a folder location and file name.
+
+![exporting](img/package-asset/exporting.png)
+
+> **Note**: Exported zip packages are only suitable for **importing resource packages** use, as described below.
+
+### Importing Assets Package
+
+Select and right click on the folder to import resources from, or right click on the **blank space** in the **Export Explorer** panel, then select **Import Package** and select the zip package exported from **Export Package** in the file browse dialog box. The resources in the zip package will be automatically parsed into the **Import Package** panel that pops up.
+
+The import process will also ask to reconfirm the imported resources. Uncheck resources to remove the unnecessary ones.
+
+![importing](img/package-asset/importing.png)
+
+> **Notes**:
+> 1. Imported zip packages, only zip packages exported by Cocos Creator and above using **Export Resource Packages** are supported.
```

Comment:
Imported zip packages, only zip packages exported by Cocos Creator and above using **Export Resource Packages** are supported.

I don't quite understand what the meaning of this is.

Reply:
The imported asset packages are only supported for zip packages exported by Cocos Creator 3.0 and above using the **Exporting Assets Package** feature.

Reply:
that's perfect @xunyi0